Editor/Bootstrap/BootstrapSettings.cs

```diff
@@ -143,6 +143,7 @@ public Response CreateBucket(string bucketName)
             }
             else
             {
+                createResponse.ErrorMessage = _textProvider.GetError(createResponse.ErrorCode);
```

Comment:
Typically the business logic for the method (in this case `CreateBucket` in BucketStore.cs) should be responsible for defining the error code and message since it knows best what's happening. Can we move this logic there instead?

Reply:
There is a problem with this, BucketStore.cs is stored in the runtime and only generates an errorcode, then it won't be until coreapi that we can translate it back so we might have to leave the translation here. 
Textprovider is editor based so we couldn't do it earlier, and the other errors in this section is API errors so they have inbuilt error messages, where as ValidationResult doesn't. We would have to hardcode the error message in there to get it back.

Reply:
I think this is one of those things that solves the problem for CreateBucket only. There are numerous other APIs (such as PutBucketPolicy etc) that could also fail, but then there's not going to be an error message that's displayed. 

I think the right way to solve this is pass down the error message from the AWS SDK call directly from BucketStore, and just display as is. This is what we're doing for GameLift service errors, and this is what we should do for other service calls as well. The full expectation is that the service returns a good enough error message. Is that possible?

Reply:
Since some paths from BucketStore do return an error message, this means we'd always be overriding that value, so I've added a check to stop this from happening. If the error messages are desired we can always remove them instead of overwriting them